ibis/bigquery/tests/test_client.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pytest
+import pandas as pd
+import pandas.util.testing as tm
+
+import ibis
+import ibis.util
+import ibis.expr.types as ir
+from ibis.bigquery import client as ibc
+from ibis.bigquery.tests import conftest as conftest
```

Comment:
Don't import conftest here. Just make `TABLE_ID` a fixture and use it as a parameter to your other fixtures and tests.